prometheus/promhttp/instrument_client.go

```diff
@@ -32,6 +32,26 @@ func (rt RoundTripperFunc) RoundTrip(r *http.Request) (*http.Response, error) {
 	return rt(r)
 }
 
+// labelsForError obtains the values of the "code" and "method" labels
+// for the case where a HTTP request failed without getting a server
+// response. The "code" label is left empty to distinguish from cases
+// where a server response was obtained.
+func labelsForError(code, method bool, reqMethod string) prometheus.Labels {
```

Comment:
Why not reusing `label` and injecting custom code (e.g -1) that would be a sentinel for us to put empty label ""? Less deduplicate functionality.